resources/aws_ec2.go

```diff
@@ -66,17 +90,85 @@ var AwsRegions = []string{
 // AWS credentials must be present in ~/.aws - For detailed instructions see
 // http://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
 type AwsEc2Res struct {
-	BaseRes `yaml:",inline"`
-	State   string `yaml:"state"`   // state: running, stopped, terminated
-	Region  string `yaml:"region"`  // region must match an element of AwsRegions
-	Type    string `yaml:"type"`    // type of ec2 instance, ie. t2.micro
-	ImageID string `yaml:"imageid"` // imageid must be available on the chosen region
+	BaseRes         `yaml:",inline"`
+	State           string `yaml:"state"`           // state: running, stopped, terminated
+	Region          string `yaml:"region"`          // region must match an element of AwsRegions
+	Type            string `yaml:"type"`            // type of ec2 instance, ie. t2.micro
+	ImageID         string `yaml:"imageid"`         // imageid must be available on the chosen region
+	WatchEndpoint   string `yaml:"watchendpoint"`   // the public url endpoint to subscribe to events, ie. http://server:12345/
+	WatchListenAddr string `yaml:"watchlistenaddr"` // the local port connected to the above url, ie. 23456
```

Comment:
s/ the local port/the local listen string/ ?? or similar (the comment)

huge nit: pick `ie:` `i.e.:` or `ex:` `ie.`, isn't a thing, eg: https://en.wiktionary.org/wiki/i.e.